Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+
+Hi there! This guide aims to teach people unfamiliar with Github how to suggest edits to the [Tanzu Agile Practices site](https://tanzu.vmware.com/developer/practices/). Follow the steps below to get started!
+
+
+# Workflow overview
+
+It takes many clicks to suggest an edit, but these are the 5 big stages you have to go through.
+
+1. **Prepare**: Type up your suggested edits 
+1. **Fork**: Forking just means creating a copy of the practices site code for your GitHub account
+1. **Edit**: Copy and paste suggested edits into the code in your copy of the practices site through the GitHub UI and save it in a branch
+1. **Submit a Pull Request**: Submitting a Pull Request is the technical term for submitting the suggested edits to be reviewed, approved by someone else and updated on the site. It is so named because the reviewer will be 'pulling' your code into the vmware repository to merge it. 
+1. **Delete**: When the change is approved, delete your copy of the practices site.
+
+Repeat steps when you want to suggest new edits to the site.
+
+# Step by Step Instructions 
+
+## 1. Prepare
+1. Type up the changes you want to make for the Practices site. Any word editor will do for this, Microsoft Word, Google Docs, Notes on Mac. For your first suggestion, try adding something small, such as a few sentences. This will make it easier for the suggestion to be approved and merged.
+1. Create a Github account, if you don’t have one. Login to your Github account, if you do.
+1. If you’re doing this for the first time, set aside around 35 minutes to go through these steps.
+
+## 2. Fork (Create a copy of Practices site)
+1. Go to the [VMware practices repository on Github](https://github.com/vmware-tanzu/tanzu-dev-portal/tree/main/content/practices) - this is where all the code for the site sits. It is also open to the public for contribution. 
+1.  Click on fork to make a copy of the code to your own github account.
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork2.png" />
```

Comment:
Is there a reason to prefer `<img>` tags over  a markdown-style image `![]()`?

Reply:
Here is a reference to [GitHub style Markdown](https://guides.github.com/features/mastering-markdown/)